packages/botonic-react/src/components/button.jsx

```diff
@@ -79,7 +79,7 @@ export const Button = props => {
       ? getThemeProperty('button.hoverBackground', COLORS.CONCRETE_WHITE)
       : getThemeProperty('button.style.background', COLORS.SOLID_WHITE)
     const buttonTextColor = hover
-      ? getThemeProperty('button.hoverText', COLORS.SOLID_BLACK)
+      ? getThemeProperty('button.hoverTextColor', COLORS.SOLID_BLACK)
```

Comment:
please take the opportunity to use the constants from botonic/packages/botonic-react/src/constants.js (where the value is correct)

Reply:
Done it @dpinol! 
Yes, better using constants. I've also changed the other `getThemeProperty` calls whithin `button.jsx` and now they are using constants too.